modules/deploy/pages/deployment-option/self-hosted/manual/sizing.adoc

```diff
@@ -14,6 +14,7 @@ In general, choose the number of nodes based on the following criteria, and add
 
 * Total network bandwidth must account for maximum simultaneous writes and reads, multiplied by the replication factor.
 * Have a minimum of 2 GB memory for each core, but more memory is better.
+* Have a minimum of 4 MB of memory for each topic partition. You can enforce this requirement in the tunable xref:reference:tunable-properties.adoc#topic_memory_per_partition[`topic_memory_per_partition` property].
```

Comment:
> 4 MB of memory for each topic partition.

See https://vectorizedio.atlassian.net/wiki/spaces/CORE/pages/99844097/Scale+Limits+in+Redpanda. It applies to partition replicas, therefore it should be

> 4 MB of memory for each topic partition replica.